src/lib.rs

```diff
@@ -244,19 +246,23 @@ impl std::error::Error for Error {}
 pub struct ThreadPriorityValue(u8);
 impl ThreadPriorityValue {
     /// The maximum value for a thread priority.
-    pub const MAX: u8 = 99;
+    pub const MAX: u8 = if cfg!(target_os = "vxworks") { 255 } else { 99 };
```

Comment:
I'm just curious: why did this change? I made it limited to exactly the same `u8` limits so that the user won't really be confused when thinking about the actual range for it, even though I also provided a way to retrieve this range, and it is the same u8 limit range. Was it necessary to lower it to `99` for VxWorks? Can't it be mapped as well as for all the other platforms into the needed (eventually) values?

Reply:
Due to these limit of 99( ideal for unix), https://github.com/iddm/thread-priority/blob/1135f2ec40d890884b29486d3b15b567eca44a49/tests/unix.rs#L151 the test failed here as tryconvert returned a error. So I had to change the limits to 255 for VxWorks.

Reply:
Due to these limit of 99( ideal for unix), I tryconvert for ThreadPriority would return error for priority values higher than 99, wheres the maximum thread priority in VxWorks is 255. So it was necessary to change the limits to 255 for VxWorks.
https://github.com/iddm/thread-priority/blob/1135f2ec40d890884b29486d3b15b567eca44a49/tests/unix.rs#L151